versioned_docs/version-8.7/self-managed/identity/application-user-group-role-management/applications.md

```diff
@@ -0,0 +1,82 @@
+---
+id: applications
+title: "Applications"
+sidebar_label: "Applications"
+---
+
+In Identity, an Application represents an entity that can request Identity to authenticate a user or a service.
+
+For Camunda 8 to work, a set of Applications are preconfigured. These preconfigured applications are:
+
+- Connectors
+- Identity
+- Operate
+- Optimize
+- Tasklist
+- Web Modeler
+- Zeebe
+
+As a user of Identity, you can add own Applications.
```

Comment:
```suggestion
As a user of Identity, you can add your own Applications.
```

I'd like some brief statement why I should do that. e.g. "... to provide a service with m2m access to a Camunda resource"

Reply:
Added.